ssn/ssn_separated/ssn.ttl

```diff
@@ -949,7 +826,7 @@ ssn:SurvivalRange rdf:type owl:Class ;
                                   ] ;
                   dc:source """skos:narrowerMatch 'limiting operating condition' [VIM 4.10]
                                   http://www.bipm.org/utils/common/documents/jcgm/JCGM_200_2008.pdf""" ;
-                  rdfs:comment "The conditions a sensor can be exposed to without damage: i.e., the sensor continues to operate as defined using MeasurementCapability.  If, however, the SurvivalRange is violated, the sensor is 'damaged' and MeasurementCapability specifications may no longer hold." ;
+                  rdfs:comment "The conditions a Sensor can be exposed to without damage: i.e., the Sensor continues to operate as defined using the MeasurementCapability. If, however, the SurvivalRange is violated, the Sensor is 'damaged' and MeasurementCapability specifications may no longer hold." ;
```

Comment:
The conditions a System can be exposed to without damage: i.e., the System continues to operate as defined using the MeasurementCapability. If, however, the SurvivalRange is violated, the System is 'damaged' and MeasurementCapability specifications may no longer hold.